lib/node_modules/@stdlib/repl/lib/completer_preview.js

```diff
@@ -199,6 +202,10 @@ setNonEnumerableReadOnly( PreviewCompleter.prototype, 'beforeKeypress', function
 	if ( !key || this._preview === '' ) {
 		return;
 	}
+	// When the user is naviating completions, don't preview completions...
+	if ( this.repl._isNavigatingCompletions ) {
```

Comment:
This is leaky and introduces tight(er) coupling between the REPL instance and the preview completer. We should use a design pattern here or move this check back up into the REPL instance.